sql/moz-fx-data-shared-prod/addon_moderations_derived/cinder_decisions_raw_v1/schema.yaml

```diff
@@ -17,7 +17,7 @@ fields:
 
 - mode: NULLABLE
   name: created_at
-  type: STRING
+  type: TIMESTAMP
```

Comment:
Anna just updated this as well as a couple other columns in https://github.com/mozilla/bigquery-etl/pull/6896, so if you merge `main` into this branch GitHub should figure that out.